02LessonTwo/homework/Homework.html

```diff
@@ -0,0 +1,101 @@
+  <option value="audi">alergies</option>
+  <option value="audi">cold</option>
+  <option value="audi">loosing</option>
+  <option value="audi">Audi</option>
+  <option value="audi">Audi</option>
+  <option value="audi">Audi</option>
+</select>
+</div>
+  <div class="topright">
+    <h3>Favorite websites</h3>
+    <table cellspacing="30px">
+      <td><a href="https://www.ted.com/talks"><img src="http://www.brainscape.com/blog/wp-content/uploads/2011/10/ted-talks-language-learning.png"alt="ted" style="width:100px;height:100px;border-radius:5px"/></td>
+      <td><a href="http://www.vice.com/en_us"><img src="https://upload.wikimedia.org/wikipedia/en/thumb/c/c0/Vice_logo.svg/1024px-Vice_logo.svg.png"alt="ted" style="width:100px;height:100px;border-radius:5px"/></td>
+      <td><a href="https://www.tumblr.com/"><img src="https://secure.assets.tumblr.com/images/logo_page/img_logo_34465d_2x.png"alt="ted" style="width:100px;height:100px;border-radius:5px"/></td>
+      <td><a href="https://twitter.com/"><img src="https://g.twimg.com/Twitter_logo_blue.png"alt="ted" style="width:100px;height:100px;border-radius:5px"/></td>
+
+      </td>
+    </table>
+
+    <table align="center"; border="2px"; cellspacing="10px">
+      <th colspan="4">
+        <h3>My Favorite Artist.</h3>
+      </th>
+      <tbody>
+        <tr>
+          <td> <strong>HIP HOP</strong> </td>
+          <td> <strong>COUNTRY</strong> </td>
+          <td> <strong>R&B</strong> </td>
+          <td> <strong>ROCK</strong></td>
+        </tr>
+        <tr>
+          <td>Drake</td>
+          <td>Kenny Chesney</td>
+          <td>R kelly</td>
+          <td>Pantera</td>
+        </tr>
+        <tr>
+          <td>Joey Badass</td>
+          <td>George Straight</td>
+          <td>Jaime Foxx</td>
+          <td>Pink Floyd</td>
+        </tr>
+        <tr>
+          <td>J Cole</td>
+          <td>Luke Bryan</td>
+          <td>Boyz To Man</td>
+          <td>The Beatles</td>
+        </tr>
+      </tbody>
+    </table>
+
+  </div>
+  <div class="bottom">
+    <h3>Movies</h3>
+    <table>
+      <td><a href="https://www.youtube.com/watch?v=G6c1HWWspGo"><img src="http://www.reallyareyouserious.com/wp-content/uploads/2015/06/Independence-Day-Movies-Forrest-Gump.png"alt="forest" style="width:420px;height:300px;border-radius:10px"/></td>
+      <td><a href="https://www.youtube.com/watch?v=RQSmfzfg2MY"><img src="http://cdn3.gbtimes.com/cdn/farfuture/Mi_FbKUe9PiQ9YB26WYTS-RSe9vUEfGBxUeh0rRYXKk/mtime:1417532777/sites/default/files/styles/1280_wide/public/2014/12/02/the-shawshank-redemption.jpg?itok=1RpNyipr"alt="ted" style="width:420px;height:300px;border-radius:10px"/></td>
+    <td><a href="https://www.youtube.com/watch?v=tQBRH1Tdk-k"><img src="https://upload.wikimedia.org/wikipedia/en/b/b6/The_Revenant_2015_film_poster.jpg"alt="ted" style="width:420px;height:300px;border-radius:10px"/></td>
+    </table>
+  </div>
+  <div class="footer"></div>
+
+</body>
+</html>
```

Comment:
Good job Larry! I liked how you separated your content by adding the blue backgrounds for each section and your use of tables for your favorite artists and movies.